diff --git a/node_modules/react-code-blocks/src/components/CopyBlock.tsx b/node_modules/react-code-blocks/src/components/CopyBlock.tsx
index e73c43b..130dc18 100644
--- a/node_modules/react-code-blocks/src/components/CopyBlock.tsx
+++ b/node_modules/react-code-blocks/src/components/CopyBlock.tsx
@@ -48,6 +48,7 @@ const Button = styled.button<Props>`
   &:focus {
     outline: none;
     opacity: 1;
+    box-shadow:0 0 0 4px #db0a5b;
   }
   .icon {
     width: 1rem;
